libpod/options.go

```diff
@@ -115,19 +115,6 @@ func WithStorageConfig(config storage.StoreOptions) RuntimeOption {
 	}
 }
 
-// WithDefaultTransport sets the default transport for retrieving images.
-func WithDefaultTransport(defaultTransport string) RuntimeOption {
```

Comment:
I don't think this is actually honored anymore (@vrothberg can confirm). We might want to fix it, though, it does live in containers.conf and setting it there has no effect.

Reply:
This is not honored anymore, correct.  I am not sure if we should continue supporting it.

Reply:
If we don't want it it should be pulled from c/common as well. Still got a few more weeks of 4.0 prep to make breaking changes.